### ISSUE TITLE
disable-condition setting nametags to a blank white name when involved with invisible-nametags

### ISSUE THREAD

**Issue by baailey** [NEZNAMY/TAB#1093]
### Server version

git-Paper-196 (MC: 1.20.1)

### TAB version

4.0.6

### Describe the bug

When you enable invisible-nametags and set the disable-condition to be %world%!=yourdesiredworld (for my case), it resets the nametag for players to be either a simple white name, or a red name when they are not in the world the nametags are invisible in https://i.imgur.com/lm4H9GW.png

### Expected behavior

When the disable-condition is met, I expect it to disable invisible-nametags and set the nametag to be what it usually is

### Steps to reproduce

1. set invisible-nametags to true
2. set disable-conditions to %world%!=yourdesiredworld
3. go to a world where nametags are visible

### Checklist

- [X] I am running latest version of the plugin
- [X] I have read the wiki to make sure it's not an issue with configuration
- [ ] I ticked all of the boxes without actually reading them
- [X] I have read the Compatibility wiki page and am not trying to run the plugin on an unsupported server version / platform

**Comment by NEZNAMY** [comment]
[Unable to reproduce](https://www.youtube.com/watch?v=1aGkSoasVhs).

**Comment by baailey** [comment]
That's fun. For the sake of it, could you try adding a custom name tag to see if it keeps your customisation or if it sets it to the basic white name you've shown in that testing?

**Comment by NEZNAMY** [comment]
I did add a tagprefix, but it was not visible due to invisible nametags being enabled.

**Comment by baailey** [comment]
So invisible nametags hides the custom name tag? is there nothing that can be done to allow the custom name to be visible when in a world invisible name tags isn't enabled in?

**Comment by NEZNAMY** [comment]
Invisible nametags is a global setting that affects all worlds.

**Comment by baailey** [comment]
is there any way i can disable _only_ invis nametags?

**Comment by NEZNAMY** [comment]
No, that's not available.

**Comment by baailey** [comment]
would you consider it as a feature?

**Comment by NEZNAMY** [comment]
No, feature requests are permanently closed as I am quitting the development.
